resources/europe/greece/gr-mailinglist.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "gr-mailinglist",
+  "type": "mailinglist",
+  "locationSet": {"include": ["el"]},
```

The [ISO 3166-1 code](https://en.wikipedia.org/wiki/List_of_countries_by_United_Nations_geoscheme) for Greece is actually `gr`.. The Greek language code is `el`.
(`el` does actually work here - which is nice, but we should use the correct code)

Can you change these in the files and rebuild?
```suggestion
  "locationSet": {"include": ["gr"]},
```

---

Yes, you are right. `el` is actually only the lang code. Commit amended.

---

@bhousel did you had the time to check the amendment?

---

looks good, thanks for the ping 👍